packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useRef } from 'react'
+import uuid from 'uuid'
+import classNames from 'classnames'
+import TetherComponent from 'react-tether'
+
+import {
+  useDatepicker,
+  FocusedInput,
+  START_DATE,
+} from '@datepicker-react/hooks'
+
+import DatepickerContext from './datepickerContext'
+
+import { Input } from './Input'
+import { Month } from './Month'
+import { useOpenClose } from './useOpenClose'
+import { DATEPICKER_PLACEMENT, DATEPICKER_PLACEMENTS } from './constants'
+import { FloatingBox, FLOATING_BOX_SCHEME } from '../../primitives/FloatingBox'
+
+export interface StateObject {
+  startDate: Date | null
```

Comment:
If `startDate` or `endDate` are nullable ie `startDate?`, is the ` | null` required?